api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -146,8 +147,9 @@ public String getDateTime(List<FieldValue> row, int index) {
     if (row.get(index).isNull()) {
       return null;
     }
-    DateTimeFormatter df = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss zzz").withZoneUTC();
-    return df.print(row.get(index).getTimestampValue() / 1000L);
+    DateTimeFormatter df =
+        DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss zzz").withZone(ZoneOffset.UTC);
+    return df.format(FieldValues.getInstant(row.get(index)));
```

Comment:
`FieldValues.getInstant()` is a new method which handles the fact that BQ uses epoch **micros** not millis (hence the conversion in the old version)

Reply:
Extracted this to #6081